wasm/richards/richards.c

```diff
@@ -16,9 +16,9 @@
 #include <stdlib.h>
 
 #if 1
-#define                Count           10000*100
-#define                Qpktcountval    2326410
-#define                Holdcountval     930563
+#define                Count           200000
+#define                Qpktcountval    465212
+#define                Holdcountval    186084
```

Comment:
I wonder if these should be parameters in the `benchmark.js` file rather than hardcoded? It probably wouldn't make a difference for perf and would make it easier to change.

Reply:
Let's do this in a follow up if we want to.

Reply:
Sorry, didn't get to it this week but can upload a PR with this next one.